test/common/expect_failure.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Simple bash script that will execute another process, which is *expected* to fail;
+# this is useful mainly for running test/error and other expected-to-fail tests.
+#
+
+echo Running $1
+
+$1
```

Comment:
Pretty sure you want quotes around this in case there are spaces in the pathname.

Reply:
Fixed